prow/cmd/deck/static/pr/pr.ts

```diff
@@ -803,6 +809,7 @@ function createQueryDetailsBtn(query: ProcessedQuery): HTMLTableDataCellElement
     const allRequired = document.getElementById("query-all-required")!;
     const allForbidden = document.getElementById("query-all-forbidden")!;
     iconButton.addEventListener("click", () => {
+        fillDetail("#query-detail-author", query.author);
```

Comment:
Doesn't this need a corresponding template change?

Reply:
You are correct; updated.